prepare/metrics/rag_answer_correctness.py

```diff
@@ -49,117 +49,128 @@ def test_answer_correctness(task_data, catalog_name, global_target, instance_tar
     )
     add_to_catalog(metric, new_catalog_name, overwrite=True)
 
-# don't use "A" as a token because it is considered an article and removed by the token overlap
-# metric
-task_data = [
-    {  # recall is 0.5 for the first ground_truth, 0 for the second ground_truth.
-        # so overall its max(0.5, 0) = 0.5
-        "ground_truths": ["B C", "C"],
-        "answer": "B",
-    },
-    {  # recall is 1/3
-        "ground_truths": ["D E F"],
-        "answer": "B C D",
-    },
-]
-
-recall_instance_targets = [
-    {"f1": 0.67, "precision": 1.0, "recall": 0.5, "score": 0.5, "score_name": "f1"},
-    {"f1": 0.33, "precision": 0.33, "recall": 0.33, "score": 0.33, "score_name": "f1"},
-]
-
-recall_global_target = {
-    "f1": 0.5,
-    "f1_ci_high": 0.67,
-    "f1_ci_low": 0.33,
-    "precision": 0.67,
-    "precision_ci_high": 1.0,
-    "precision_ci_low": 0.33,
-    "recall": 0.42,
-    "recall_ci_high": 0.5,
-    "recall_ci_low": 0.33,
-    "score": 0.42,
-    "score_ci_high": 0.67,
-    "score_ci_low": 0.33,
-    "score_name": "f1",
-}
-
-
-for catalog_name, global_target, instance_targets in [
-    ("metrics.rag.answer_correctness", recall_global_target, recall_instance_targets),
-    ("metrics.rag.recall", recall_global_target, recall_instance_targets),
-]:
-    test_answer_correctness(task_data, catalog_name, global_target, instance_targets)
-
+if __name__ == "__main__":
+    # don't use "A" as a token because it is considered an article and removed by the token overlap
+    # metric
+    task_data = [
+        {  # recall is 0.5 for the first ground_truth, 0 for the second ground_truth.
+            # so overall its max(0.5, 0) = 0.5
+            "ground_truths": ["B C", "C"],
+            "answer": "B",
+        },
+        {  # recall is 1/3
+            "ground_truths": ["D E F"],
+            "answer": "B C D",
+        },
+    ]
 
-test_answer_correctness(
```

Comment:
How is this change related?

Reply:
could be some merge that got it

Reply:
Yotam did it when working on rag

Reply:
I saw another PR onthese metric yesterday.  Can you review with  @perlitz - that these change is correct.

I want to make sure we don't revert / make unintended changes.